xlators/nfs/server/src/nfs.h

```diff
@@ -101,6 +101,10 @@ struct nfs_state {
     char *rpc_statd_pid_file;
     gf_boolean_t rdirplus;
     uint32_t event_threads;
+
+#ifdef HAVE_LIBTIRPC
+    bool svc_running;
```

Comment:
nit: it's not like the structure is anyway aligned, but this is not helping ;-)

Reply:
Why not ? I've already put it the last field of the structure. A `bool` is a single byte type. All other types (including `gf_boolean_t`) are at least 4 bytes. I don't see how I can improve the layout of the structure without reordering all fields.

Reply:
I thought attaching it to another boolean would have helped a bit.

Reply:
I've just realized that `gf_boolean_t` is now an alias of `bool`. Previously it was an `enum`. In this case, yes, I could reorder the fields a bit. If I do that, wouldn't it be better to reorder all fields instead of only bools ?

Reply:
I would say yes, of course. If we are already touching a structure, let's do the janitor work and fix alignment. Others may disagree. But it's a nit, really, so no strong feels either way.